src/main/java/core/basesyntax/transactions/TransactionList.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.transactions;
+
+import core.basesyntax.parsers.ParseFromFile;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionList {
```

Comment:
let's rename the class that reads lines from file `FileReaderService`

then rename this class `TransactionParser`